variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "bucket_name" {
+  type        = string
+  description = "Name to be used for GCS bucket to which Worklytics data will be exported (eg 'acme-co-worklytics-export')."
+}
+
+variable "worklytics_tenant_sa_email" {
+  type        = string
+  description = "Email address of your Worklytics tenant's service account (obtain from Worklytics App)."
+}
+
+variable "worklytics_host" {
+  type        = string
+  description = "host of worklytics instance where tenant resides. (e.g. intl.worklytics.co for prod; but may differ for dev/staging)"
+  default     = "intl.worklytics.co"
```

Comment:
`.app` vs. `.intl`, shouldn't we agree on always using the same?

Reply:
yeah ... that's a question - we should depend on `app` to do the redirect??